scripts/widgets/mz-search-pagination-dd.js

```diff
@@ -20,7 +20,11 @@ define(["jquery", "underscore"], function ($, _) {
             "(": "^(",
             ")": "^)"
         };
-        var dataSet = new options.model({ pageSize: options.pageSize, filter: options.filterKey + ' ' + options.filterOption1 + ' ' + options.filterValue, sortBy: options.textField + ' ' + 'asc' });
+        var filterUserIdStr = "";
+        if (options.userId) {
+            filterUserIdStr += "salesrep.userid" + ' ' + options.filterOption1 + ' ' + options.userId + " and ";
+        }
+        var dataSet = new options.model({ pageSize: options.pageSize, filter: filterUserIdStr +  options.filterKey + ' ' + options.filterOption1 + ' ' + options.filterValue, sortBy: options.textField + ' ' + 'asc' });
```

Comment:
This is a generic component so we can't hardcode anything inside it. Make your changes using the config options for the component. You will need to revert all your changes in this file.

Reply:
Done.